docs/parallel_config.md

```diff
@@ -0,0 +1,114 @@
+## Parallel workflow configuration class
+
+`WorkflowConfig` is a class that stores parallel workflow configuration parameters. Configurations can be saved/imported
+to run workflows in parallel.
+
+*class* **plantcv.parallel.WorkflowConfig**
+
+**Class methods**
+
+Save a configuration file that can be modified and imported to run workflows in parallel.
+
+**WorkflowConfig.save_config**(*config_file*)
+
+- **Parameters:**
+    - config_file (str, required): path/name of output configuration file
+- **Context:**
+    - Used to create a configuration file that can be edited and imported
+
+Import a configuration file.
+
+**WorkflowConfig.import_config**(*config_file*)
+
+- **Parameters:**
+    - config_file (str, required): path/name of input configuration file
+- **Context:**
+    - Used to import configuration settings from a file
+
+Validate parameters/structure of configuration data.
+
+**WorkflowConfig.validate_config**()
+
+- **Parameters:**
+    - None
+- **Context:**
+    - Used to run validation checks on configuration settings
+
+- **Parameters:**
+    - input_dir (str, required): path/name of input images directory (validates that it exists)
+    - json (str, required): path/name of output JSON data file (appends new data if it already exists)
+    - filename_metadata (list, required): list of metadata terms used to construct filenames
```

Comment:
We should have an example of the formatting needed here. This takes a list of strings (in quotes) which should match up with valid metadata terms.